### PR TITLE
Added support for 3d_segmentation_demo validation

### PR DESCRIPTION
**Pull request by eizamaliev**
Added visual comparison with annotation (use key `--path_to_ground_truth`) and dice index calculation.

TODO:
- [ ] Update readme
- [ ] Add support for setting label order

### CODE REVIEW

**Comment by eizamaliev** [comment]
cc @elenash @vladimir-dudnik

File: demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py [hunk 1]
```diff
@@ -347,10 +381,30 @@ def main():
 
         im = 255 * (im - im.min())/(im.max() - im.min())
         color_seg_frame = np.zeros(im.shape, dtype=np.uint8)
-        for idx, c in enumerate(CLASSES_COLOR_MAP):
+        class_order = (0, 2, 1, 3)
```
Comment (elenash) [inline demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py]: Why it's hardcoded here?

Comment (eizamaliev) [reply demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py]: @elenash please be patient. This item is in TODO list (see in PR description)

**Comment by IRDonch** [comment]
Are you still working on this?

**Pull request closed without merging**